### PR TITLE
feat: 0.14.0

### PR DESCRIPTION
**Pull request by nandi95**

### CODE REVIEW

File: src/DemoBoard.vue [hunk 1]
```diff
@@ -57,7 +57,7 @@
                     <h1 v-if="$route.meta.label" class="text-2xl mb-2 text-center text-color">
                         {{ $route.meta.label }}
                     </h1>
-                    <router-view />
+                    <RouterView />
```
Comment (fullstackfool) [inline src/DemoBoard.vue:60]: 🙏

File: src/components/table/UITable.vue [hunk 2]
```diff
@@ -361,13 +360,13 @@ export default defineComponent({
     setup(props, ctx) {
         const chevronIcon = getIcon('chevron');
         let styleTagId = '';
-        const itemPerPageOptions = [
+        const itemPerPageOptions = markRaw([
```
Comment (fullstackfool) [inline src/components/table/UITable.vue:363]: What is the intention of `markRaw` being used for `itemPerPageOptions`?

**Review by fullstackfool — APPROVED** [review]

File: src/components/table/UITable.vue (hunk 2 above)
Comment (nandi95) [reply src/components/table/UITable.vue:363]: Performance improvement (and perhaps readibility)
https://vuejs.org/api/reactivity-advanced.html#markraw
> ...allow you to selectively opt-out of the default deep reactive/readonly conversion and embed raw, non-proxied objects in your state graph

Comment (fullstackfool) [reply src/components/table/UITable.vue:363]: Oh I know what it does, but I'm confused about the performance aspect for a small, hard-coded, never-changing internal array.

No downside to having it there, just can't see the upside in this particular instance.

**Pull request merged**